examples/notebook/pytorch/Quick_Started_Notebook_of_INC_for_Pytorch.ipynb

```diff
@@ -0,0 +1,342 @@
+    "    eval_dataloader = _build_inc_dataloader(eval_dataloader)\n",
+    "batch_size = eval_dataloader.batch_size\n",
+    "\n",
+    "def take_eval_steps(model, trainer, save_metrics=False):\n",
+    "    trainer.model = model\n",
+    "    metrics = trainer.evaluate()\n",
+    "    bert_task_acc_keys = ['eval_f1', 'eval_accuracy', 'eval_matthews_correlation',\n",
+    "                            'eval_pearson', 'eval_mcc', 'eval_spearmanr']\n",
+    "    for key in bert_task_acc_keys:\n",
+    "        if key in metrics.keys():\n",
+    "            throughput = metrics.get(\"eval_samples_per_second\")\n",
+    "            print('Batch size = %d' % batch_size)\n",
+    "            print(\"Finally Eval {} Accuracy: {}\".format(key, metrics[key]))\n",
+    "            print(\"Latency: %.3f ms\" % (1000 / throughput))\n",
+    "            print(\"Throughput: {} samples/sec\".format(throughput))\n",
+    "            return metrics[key]\n",
+    "    assert False, \"No metric returned, Please check inference metric!\"\n",
+    "\n",
+    "def eval_func(model):\n",
+    "    return take_eval_steps(model, trainer)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2 Run Quantization\n",
+    "\n",
+    "So far, we can finally start to quantize the model. \n",
+    "\n",
+    "To start, we need to set the configuration for post-training quantization using `PostTrainingQuantConfig` class. Once the configuration is set, we can proceed to the next step by calling the `quantization.fit()` function. This function performs the quantization process on the model and will return the best quantized model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from neural_compressor.quantization import fit\n",
+    "from neural_compressor.config import PostTrainingQuantConfig, TuningCriterion\n",
+    "tuning_criterion = TuningCriterion(max_trials=600)\n",
+    "conf = PostTrainingQuantConfig(approach=\"static\", tuning_criterion=tuning_criterion)\n",
+    "q_model = fit(model, conf=conf, calib_dataloader=eval_dataloader, eval_func=eval_func)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Benchmark with Intel® Neural Compressor\n",
+    "\n",
+    "INC provides a benchmark feature to measure the model performance with the objective settings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from neural_compressor.config import BenchmarkConfig\n",
+    "from neural_compressor import benchmark\n",
+    "b_conf = BenchmarkConfig(warmup=5,\n",
+    "                        iteration=100,\n",
+    "                        cores_per_instance=4,\n",
+    "                        num_of_instance=1)\n",
+    "# fp32\n",
+    "benchmark.fit(model, b_conf, b_dataloader=eval_dataloader)\n",
```

Comment:
This is not the correct way for benchmark. It should be failed with numactl issue in the first trigger and pass when you trigger it again right? You can refer to the tensorflow example to change the benchmark method, onnxruntime example also need to update, I have already sync with the example owner.